runtime/doc/debugger.txt

```diff
@@ -119,7 +119,6 @@ The following compile time preprocessor variables control the features:
     Debug Glyphs				FEAT_SIGNS
     Debug Highlights				FEAT_SIGNS
     Message Footer				FEAT_FOOTER
-    Balloon Evaluation				FEAT_BEVAL
```

Comment:
> I wasn't sure if I can delete all the 2. Vim Compile Options *debugger-compilation* paragraph . Should I do this in a commit ?

Good idea, but do you mind If instead I do that in #2532? That PR is mostly about documentation while this one already has a pretty specific focus (cleaning up macros). I'll be sure to credit your in the commit message, but it's up to you.